scripts/foreach.sh

```diff
@@ -41,52 +44,78 @@ function success {
   printf "\e[32;5;81m$@\e[0m\n"
 }
 
-if [[ "${1:-}" == "--reset" ]]; then
-    rm -f "${statedir}/.foreach."*
-    success "state cleared. you are free to start a new command."
-    exit 0
+function reset {
+  rm -f "${statedir}/.foreach."*
+  success "state cleared. you are free to start a new command."
+}
+
+DIRECTION=""
+RESET=0
+FORCE=0
+SKIP=0
+command_arg=""
+
+for arg in "$@"
+do
+  case "$arg" in 
+    -r | --reset) RESET=1               ;;
+    -s | --skip)  SKIP=1                ;;
+    -f | --force) FORCE=1               ;;
```

Comment:
I think having the `--reset` gives us some flexibility when we go to make our own scripts. For example, the `builddown` and `buildup` script both use `--reset` and it helps provide other people access to clear the queue. If anything I think the `--force` flag is more for instant configuration while `--reset` gives us the ability to run some actions in between calls to `foreach.sh`

Reply:
I probably was not clear on my previous comment. I'm not suggesting to remove the `reset` flag or changing its behaviour - let's keep it right where it is.

I'm suggesting dropping the `force` flag and simply reusing `reset` to do the same functionality. So instead of -

```
foreach.sh --force --up yarn build
```

one would do

```
foreach.sh --reset --up yarn build
```

Reply:
Ooo okay I see. Yeah this sounds better because if someone is using `--reset` and `--DIRECTION` then its pretty implicit that it would reset and commit an action in the right direction. 

I can make the change right now!